index.html

```diff
@@ -53,6 +53,230 @@ <h1>I'm Anold. <br>Glad to see <br> you!</h1>
+        <h2>Tonic</h2>
+        <div class="canopy">
+          <div>
+            <h5>CANOPY</h5>
+          </div>
+          <ul>
+            <li>
+              <img src="images/Counter.png" alt="counter image">
+              <span>Backend dev</span>
+            </li>
+            <li>
+              <img src="images/Counter.png" alt="counter image">
+              <span>2015</span>
+            </li>
+          </ul>
+        </div>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="languages">
+          <li>HTML</li>
+          <li>CSS</li>
+          <li>JavaScript</li>
+        </ul>
+        <button type="button">See Project</button>
+      </article>
+      <article class="work">
+        <img src="images/Snapshoot Portfolio.png" alt="">
+        <h2>Tonic</h2>
+        <div class="canopy">
+          <div>
+            <h5>CANOPY</h5>
+          </div>
+          <ul>
+            <li>
+              <img src="images/Counter.png" alt="counter image">
+              <span>Backend dev</span>
+            </li>
+            <li>
+              <img src="images/Counter.png" alt="counter image">
+              <span>2015</span>
+            </li>
+          </ul>
+        </div>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="languages">
+          <li>HTML</li>
+          <li>CSS</li>
+          <li>JavaScript</li>
+        </ul>
+        <button>See Project</button>
+      </article>
+    </section>
+
+    <section class="about_me">
+      <article class="about_myself">
+        <h1>About Myself</h1>
+        <p class="tomato">Hello I'm a software developer! I can help you build a product , feature or website. Look through some of my
+          work and
+          experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
+        <article class="social">
+          <div class="margin_sm lets_connect">
+            <a href="#">LET'S CONNECT</a>
+          </div>
+          <ul class="social_icons">
+            <li>
+              <a href="#"><img src="images/twitter.svg" alt="twitter icon"></a>
+            </li>
+            <li>
+              <a href="#"><img src="images/Linkedin icon-1.svg" alt="Linkedin icon"></a>
+            </li>
+            <li>
+              <a href="#"><img src="images/medium.svg" alt="medium icon"></a>
+            </li>
+            <li>
+              <a href="#"><img src="images/github.svg" alt="github icon"></a>
+            </li>
+            <li>
+              <a href="#"><img src="images/angellist.svg" alt="angellist icon"></a>
+            </li>
+          </ul>
+          <button class="my_resume margin_sm">Get my resume</button>
+
+          <!--technologies-->
+          <div class="technologies">
+            <ul class="technologies_container">
+              <li>
```

Comment:
Kindly use the right font styles for this section of your work, it should be similar to the design given in the figma template 😄 

Your design :
![image](https://github.com/MPDADDY/my_portfolio/assets/20537712/61d1814b-cb20-45fc-98d4-678c6de18edc)

Figma design :
![image](https://github.com/MPDADDY/my_portfolio/assets/20537712/3343abf0-00e3-4314-8f18-02a57a1946d3)